src/main/java/org/opentestsystem/ap/imrt/common/model/BaseItem.java

```diff
@@ -76,10 +76,8 @@
     @OrderBy("type ASC")
     private List<StandardId> standardIds;
 
-    @NotBlank
```

Comment:
These annotations are the problem.  They tell the hibernate validator these fields cannot be blank so hibernate was stopping this from being saved at all.  The constraint was happening within the code.

I found this issue by turning on debugging for hibernate SQL and watching the logs as I ingested an item.  I didn't see any inserts happening which I found to be strange leading me to think it was happening somewhere in the code. Removing these allows us to ingest items with blank values.